src/util/type.js

```diff
@@ -9,4 +9,14 @@ export function isDOMNode(o){
     typeof Node === "object" ? o instanceof Node : 
     o && typeof o === "object" && typeof o.nodeType === "number" && typeof o.nodeName==="string"
   );
+}
+
+// If constructor is not in object's prototype chain,
```

Comment:
Nice! Let's use JSDoc style comments for methods like this, so that if we ever generate documentation from our code, we are all good to go.

```
/**
 * Description
 *
 * @param whatever Here is a description of the whatever argument
 * @return {*} Some return description
 */
```